python/tvm/contrib/cc.py

```diff
@@ -131,14 +116,43 @@ def cross_compiler(compile_func, base_options=None, output_format="so", get_targ
     -------
     fcompile : Callable[[str, str, Optional[str]], None]
         A compilation function that can be passed to export_library.
+
+    Examples
+    --------
+    .. code-block:: python
+
+       from tvm.contrib import cc, ndk
+       # export using arm gcc
+       mod = build_runtime_module()
+       mod.export_library(path_dso,
+                          cc.cross_compiler("arm-linux-gnueabihf-gcc"))
+       # specialize ndk compilation options.
+       specialized_ndk = cc.cross_compiler(
+           ndk.create_shared, ["--sysroot=/path/to/sysroot"])
```

Comment:
options should be better changed into ["--sysroot=/path/to/sysroot", "-shared", "-fPIC", "-lm"], because when users pass options into ndk, we won't append the latter three options so that we won't compile into shared libraries.